cvpysdk/deployment/install.py

```diff
@@ -493,18 +511,27 @@ def install_software(
                                                 "overrideSoftwareCache": False,
                                                 "components": {
                                                     "commonInfo": {
-                                                        "globalFilters": 2
+                                                        "globalFilters": 2,
+                                                        "storagePolicyToUse": {
+                                                            "storagePolicyName": storage_policy_name
+                                                        }
                                                     },
                                                     "fileSystem": {
                                                         "configureForLaptopBackups": False
                                                     },
                                                     "componentInfo": install_options,
                                                 },
                                                 "clientInfo": {
+                                                    "clientGroups": [{
+                                                        "clientGroupName": client_group_name
```

Comment:
Can we set these fields only if client_group_name is not None? Similarly below as well, if log_file_loc is not None?

Reply:
Yes. By chance, do you have guidance regarding best-practice for implementing those checks? Let me know if there's a good reference model within the cvpysdk, so that I can attempt to maintain consistency within the sdk. Thanks.

Reply:
As of now, I don't have an exact example of this use case. It's more of a precautionary check. 

But yes you can refer to below coding considerations while working on sdk. 
https://github.com/Commvault/cvpysdk#coding-considerations